tests/framework/utils/k8s_helper.go

```diff
@@ -1659,3 +1662,68 @@ func (k8sh *K8sHelper) WaitForLabeledDeploymentsToBeReadyWithRetries(label, name
 	}
 	return fmt.Errorf("giving up waiting for deployment(s) with label %s in namespace %s to be ready", label, namespace)
 }
+
+// Creates a kubeconfig file and pastes it in $HOME/.kube/config
+func (k8sh *K8sHelper) MakeKubeConfig() {
+	context, err := k8sh.MakeContext().Executor.ExecuteCommandWithOutput("kubectl", "config", "view", "--raw")
+	if err != nil {
+		logger.Errorf("failed to fetch kube context. %v", err)
+	}
+	home, err := os.UserHomeDir()
+	if err != nil {
+		logger.Errorf("failed to fetch homedir. %v", err)
+	}
+	err = ioutil.WriteFile(path.Join(home, ".kube", "config"), []byte(context), os.ModePerm)
+	if err != nil {
+		fmt.Printf("failed to write to file $HOME/.kube/config. %v", err)
+	}
+
+	data, err := ioutil.ReadFile(path.Join(home, ".kube", "config"))
+	fmt.Print(string(data))
+}
+
+// Gets status for admission controllers
+func (k8sh *K8sHelper) GetStatus(namespace string) {
```

Comment:
Since you're done troubleshooting the CI we can remove this verbose output, right? This whole func can be deleted.